airo/.htaccess

```
@@ -26,7 +26,7 @@ RewriteCond %{HTTP_ACCEPT} application/ld\+json
 RewriteRule ^$ https://delaramglp.github.io/airo/airo.jsonld [R=302,L]
 
 # https://w3id.org/airo/1.0
-RewriteRule ^1.0$ https://delaramglp.github.io/airo/[R=302,L]
+RewriteRule ^1.0$ https://delaramglp.github.io/airo [R=302,L]
```

GitHub will do a 301 redirect and add the trailing `/`.  Consider updating the rule below too.
```suggestion
RewriteRule ^1.0$ https://delaramglp.github.io/airo/ [R=302,L]
```

---

Sure. Thanks!